include/Core/Config.php

```diff
@@ -132,15 +132,33 @@ public static function set($family,$key,$value) {
 		$dbvalue = (is_array($value)?serialize($value):$value);
 		$dbvalue = (is_bool($dbvalue) ? intval($dbvalue) : $dbvalue);
 
-		$ret = q("INSERT INTO `config` ( `cat`, `k`, `v` ) VALUES ( '%s', '%s', '%s' )
-ON DUPLICATE KEY UPDATE `v` = '%s'",
+		// The "INSERT" command is very cost intense. It saves performance to do it this way.
+		$ret = q("SELECT `v` FROM `config` WHERE `cat` = '%s' AND `k` = '%s' ORDER BY `id` DESC LIMIT 1",
 			dbesc($family),
-			dbesc($key),
-			dbesc($dbvalue),
-			dbesc($dbvalue)
+			dbesc($key)
 		);
+
+		// It would be better to use the dbm class.
+		// But there is an autoloader issue that I don't know how to fix:
+		// "Class 'Friendica\Core\dbm' not found"
+		//if (!dbm::is_result($ret))
```

Comment:
@fabrixxm can you write a quick tutorial for using autoloader which will be added to friendicas help manual?